test/server/cards/01_SOR/events/TheEmperorsLegion.spec.ts

```diff
@@ -0,0 +1,55 @@
+describe('The Emprerors Legion', function () {
+    integration(function (contextRef) {
+        it('should return card to player hand from a discard pile', function () {
+            contextRef.setupTest({
+                phase: 'action',
+                player1: {
+                    hand: ['the-emperors-legion'],
+                    groundArena: ['pyke-sentinel', 'seasoned-shoretrooper', 'blizzard-assault-atat'],
+                    spaceArena: ['cartel-spacer'],
+                    discard: ['keep-fighting', 'green-squadron-awing', 'disarm']
+                },
+                player2: {
+                    hand: ['superlaser-blast'],
+                    discard: ['tactical-advantage', 'guerilla-attack-pod']
+                }
+            });
+            const { context } = contextRef;
+
+            context.player1.passAction();
+            context.player2.clickCard(context.superlaserBlast);
+            expect(context.pykeSentinel.location).toBe('discard');
```

Comment:
We typically only use `expect()` to check the results of the card being tested. In this case, we don't need to check that cards correctly end up in discard from SLB since that would be more of a test of that card's implementation.